src/main/scala/com/aamend/spark/gdelt/GdeltParser.scala

```diff
@@ -298,7 +298,7 @@ object GdeltParser {
 
   private def buildPersons(str: String) = str.split(";").toList
 
-  private def buildSourceCollectionIdentifier(str: String) = sourceCollectionIdentifier(str.toInt)
+  private def buildSourceCollectionIdentifier(str: String) = Try(sourceCollectionIdentifier(str.toInt)).getOrElse(sourceCollectionIdentifier(-1))
```

Comment:
Eventually would need to move to options instead of default value. The timestamp of 0L or identifier as -1 is not the best possible practice. But can come later

Reply:
Yes, The latest commit puts the parsing error in place. This is how I figured out why several rows were parsing to the default of timestamp(0L), etc. Basically all caused by out of index errors as some of the rows og GKG don't have all the 20+ fields.